patches/server/0380-Don-t-run-entity-collision-code-if-not-needed.patch

```diff
@@ -23,7 +23,7 @@ index e6e4856db7d983460fdf898583915ff2e4a43d14..25e1dc97e3260e7f58b15a1759b8f953
 +            if (!this.isPushable()) {
 +                return;
 +            }
-+            net.minecraft.world.scores.Team team = this.getTeam();
```

Comment:
I'd probably argue that, if we are adding a new patch for this, this change should be in the added patch as well.

Fixing it in two places makes dropping a patch harder / knowing what the patch/fix entails.
Not too sure on it tho, I hope we can get some PRarathon started this weekend and I'll get back on this :+1: